eZ/Publish/Core/REST/Server/Output/ValueObjectVisitor/RestExecutedView.php

```diff
@@ -133,5 +128,39 @@ public function visit( Visitor $visitor, Generator $generator, $data )
 
         $generator->endObjectElement( 'View' );
     }
-}
 
+    private function mapSearchHit( SearchHit $searchHit )
+    {
+        if ( $searchHit->valueObject instanceof Content )
+        {
+            return $this->mapContentSearchHit( $searchHit->valueObject );
+        }
+
+        if ( $searchHit->valueObject instanceof Location )
+        {
+            return $this->mapLocationSearchHit( $searchHit->valueObject );
+        }
+    }
+
+    /**
+     * @return RestContent
+     */
+    private function mapContentSearchHit( Content $content )
+    {
+        return new RestContentValue(
+            $content->contentInfo,
```

Comment:
It does. I guess it was done this way for a reason...

I don't wanna handle it as part of this PR, but feel free to submit a PR against master that improves that aspect :-)